exploring_UV_extinction_curves_pt2/Exploring_UV_extinction_curves_pt2.ipynb

```diff
@@ -0,0 +1,746 @@
+{
```

Comment:
<p>I think there needs to be some more introduction about the fact that we are going to fit a model to the above, and what that model will be. FM 1990 is presented next, but I had to go into the paper to see what the parameterization would be. It would be good to include at least Equation 2, if not also 3 and 4, in the introduction. The text above those equations also helped me understand it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>